### ISSUE TITLE
Unknowns parameters for BatchNormalization

### ISSUE THREAD

**Issue by NicolasMICAUX** [cleverhans-lab/entangled-watermark#4]
Hello,
When migrating your code from TF1 to TF2, I cannot figure out what some parameters for batch_normalization (in resnet.py, line 153) are : `decay=0.9, updates_collections=None, scope=scope`

My research gave me the following results : 

- momentum = 1 - decay (according to https://github.com/keras-team/keras/issues/6839)
- updates_collections : is it related to parameter 'Trainable' of BatchNormalization in TF2 ?
- scope : i understand the objective of scope, but cannot understand why it is used here.

here is my (working) code without those parameters using TF2 :
```
def batch_norm(x, is_training=True, scope='batch_norm'):
    return tf_keras.layers.BatchNormalization(epsilon=1e-05,
                                              center=True, scale=True)(x, training=is_training)
```

Thx a lot,
